Superlinter tries to lint files just removed in the current commit

## Describe the bug

Superlinter tries to lint files just removed in the current commit.

## Expected behavior

Superlinter lints changed files, but not those that were removed. See also #671.

## Steps to Reproduce

1. Add two yml files.
2. Commit and push.
3. Remove one yml file.
4. Commit and push.

## Logs

```log
2020-11-10 13:55:32 [INFO]   File:[.github/workflows/Linter.yml]
2020-11-10 13:55:32 [ERROR]   Found errors in [yamllint] linter!
2020-11-10 13:55:32 [ERROR]   Error code: 255. Command run:[$[Errno 2] No such file or directory: '.github/workflows/Linter.yml']
2020-11-10 13:55:32 [INFO]   ---------------------------
2020-11-10 13:55:32 [INFO]   File:[.github/workflows/Materialen.yml]
2020-11-10 13:55:32 [INFO]    - File:[Materialen.yml] was linted with [yamllint] successfully
2020-11-10 13:55:32 [INFO]   ---------------------------
2020-11-10 13:55:32 [INFO]   File:[.github/workflows/Publiceren.yml]
2020-11-10 13:55:32 [ERROR]   Found errors in [yamllint] linter!
2020-11-10 13:55:32 [ERROR]   Error code: 255. Command run:[$[Errno 2] No such file or directory: 
```

--- a/lib/buildFileList.sh
+++ b/lib/buildFileList.sh
@@ -143,6 +143,15 @@ function BuildFileList() {
     ##############
     debug "File:[${FILE}], File_type:[${FILE_TYPE}], Base_file:[${BASE_FILE}]"
 
+    ##########################################################
+    # Check if the file exists on the filesystem, or skip it #
+    ##########################################################
+    if [ ! -f "${FILE}" ]; then
+      # File not found in workspace
+      debug "File:{$FILE} existed in commit data, but not found on file system, skipping..."
+      continue
+    fi
+
     ########################################################
     # Don't include test cases if not running in test mode #
     ########################################################
@@ -159,9 +168,9 @@ function BuildFileList() {
     # Editorconfig-checker should check every file
     FILE_ARRAY_EDITORCONFIG+=("${FILE}")
 
-    ######################
+    #######################
     # Get the shell files #
-    ######################
+    #######################
     if IsValidShellScript "${FILE}"; then
       ################################
       # Append the file to the array #

